xml/hardening.xml

```diff
@@ -4007,11 +4025,12 @@ RhostsRSAAuthentication no</screen>
     </sect2>
 
     <sect2 xml:id="sec.sec_prot.general.add_random">
-     <title>Virtual Address Space Randomization</title>
+     <title>Buffer Overflow Attack Mitigation</title>
      <para>
-      Starting with the 2.6.x kernel releases, Linux uses the address-space
-      randomization technique to mitigate buffer overflows. For more
-      information, see:
+      Starting with the 2.6.x kernel releases, Linux offers <emphasis>
```

Comment:
Try to avoid spaces after `<emphasis>`. So better remove the space between the start tag and the text.